src/main/daml/Daml/Finance/Lifecycle/Rule/Util.daml

```diff
@@ -0,0 +1,53 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Lifecycle.Rule.Util where
+
+import DA.Either (partitionEithers)
+import DA.List (head, groupOn)
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (K, Q, qty)
+
+-- | Type used to reference assets in the claim tree.
+type Deliverable = Instrument.K
```

Comment:
I think we can remove this type definition and use directly `Instrument.K` below. `Deliverable` makes sense in the context of claims